src/runtime_src/core/pcie/linux/shim.cpp

```diff
@@ -3150,3 +3172,36 @@ xclGetDebugIpLayout(xclDeviceHandle hdl, char* buffer, size_t size, size_t* size
     *size_ret = 0;
   return;
 }
+
+void
+asd_parser::
+get_aie_status_version_info(xclDeviceHandle hdl, uint16_t &aie_ver_major, uint16_t &aie_ver_minor)
```

Comment:
We are working on removing the xcl infrastructure from XRT. I would not follow the model that is set here.